docs/whats-new/2021-04.md

```diff
@@ -0,0 +1,93 @@
+- [CA1841: Prefer Dictionary Contains methods](../fundamentals/code-analysis/quality-rules/ca1841.md) - Ca1841
+- [PreserveCompilationContext not configured by default](../core/compatibility/aspnet-core/6.0/preservecompilationcontext-not-set-by-default.md) - PreserveCompilationContext breaking change
+- [SignalR: Java Client updated to RxJava3](../core/compatibility/aspnet-core/6.0/signalr-java-client-updated.md) - Java client breaking change
+- [AddDataAnnotationsValidation method made obsolete](../core/compatibility/aspnet-core/6.0/adddataannotationsvalidation-obsolete.md) - AddDataAnnotationsValidation breaking change
+- [Simplify object creation (IDE0140)](../fundamentals/code-analysis/style-rules/ide0140.md) - Document IDE0140
+- [IL3002: Avoid calling members annotated with 'RequiresAssemblyFilesAttribute' when publishing as a single file.](../fundamentals/code-analysis/quality-rules/il3002.md) - Add IL3002
+- [Prepare .NET libraries for trimming](../core/deploying/prepare-libraries-for-trimming.md) - Document new linker options and trimming libraries
+- [Razor: Compiler no longer produces a Views assembly](../core/compatibility/aspnet-core/6.0/razor-compiler-doesnt-produce-views-assembly.md) - Breaking change for razor
+- [Changed MessagePack library in @microsoft/signalr-protocol-msgpack](../core/compatibility/aspnet-core/6.0/messagepack-library-change.md) - MessagePack breaking change
+- [Assemblies removed from Microsoft.AspNetCore.App shared framework](../core/compatibility/aspnet-core/6.0/assemblies-removed-from-shared-framework.md) - Shared framework breaking change
+- [WebRequest, WebClient, and ServicePoint are obsolete](../core/compatibility/networking/6.0/webrequest-deprecated.md) - WebRequest deprecation
+
+## C# language
+
+### New articles
+
+- [is operator (C# reference)](../csharp/language-reference/operators/is.md) - Moved and revised the is operator article
+- [Patterns (C# reference)](../csharp/language-reference/operators/patterns.md) - Added C# patterns reference
+- [Unsafe code and pointer types](../csharp/language-reference/unsafe-code.md) - Create unsafe code section - include information on function pointers
+
+### Updated articles
+
+- [C# Keywords](../csharp/language-reference/keywords/index.md) - Add unmanaged and managed function pointer call keywords
+- [Recommended tags for documentation comments (C# programming guide)](../csharp/programming-guide/xmldoc/recommended-tags-for-documentation-comments.md) - C# Metadata sweep
+- [Tutorial: Make HTTP requests in a .NET console app using C\#](../csharp/tutorials/console-webapiclient.md) - Edit pass on REST tutorial
+- [What's new in C# 9.0](../csharp/whats-new/csharp-9.md) - C# 9 what's new edit pass
+- [C# Coding Conventions (C# Programming Guide)](../csharp/programming-guide/inside-a-program/coding-conventions.md) - Move code comments to article text
```

Comment:
@tdykstra Do any of these meet the bar in your mind?

Reply:
No, I would omit from the docs what's new; most are primarily refactoring existing text, not introducing much new information.

Reply:
Done.